webknossos/webknossos/dataset/_utils/pims_images.py

```diff
@@ -510,15 +502,13 @@ def copy_to_view(
                 for image_slice in images[z_start:z_end]:
                     image_slice = np.array(image_slice)
                     # place channels first
-                    if self._img_dims.endswith("c"):
-                        image_slice = np.moveaxis(image_slice, source=-1, destination=0)
-                    # ensure the last two axes are xy:
-                    if ("yx" in self._img_dims and not self._swap_xy) or (
-                        "xy" in self._img_dims and self._swap_xy
-                    ):
-                        image_slice = image_slice.swapaxes(-1, -2)
-
-                    if "c" in self._img_dims:
+                    if "c" in self._bundle_axes:
```

Comment:
As the BoundingBox currently doesn't know about the channel axis, I move the channel axis always to the first position. If it is already at first position, nothing will change. But if it is between `xyz` and additional axes, it will still move to the first position.